dev/com.ibm.ws.classloading_sharedclass_fat/publish/servers/sharedClassesEarTest/bootstrap.properties

```
@@ -0,0 +1,2 @@
+bootstrap.include=../testports.properties
+com.ibm.ws.logging.trace.specification=*=info:com.ibm.ws.classloading.internal.AppClassLoader=debug:com.ibm.ws.classloading.internal.ContainerClassLoader=debug
```

```suggestion
com.ibm.ws.logging.trace.specification=*=info:com.ibm.ws.classloading.internal.AppClassLoader=finest:com.ibm.ws.classloading.internal.ContainerClassLoader=finest
```
`finest` is the new way of saying `debug`.